client/src/components/generic/Header.js

```diff
@@ -85,7 +85,10 @@ export const Header = ({ hideEmployers = false }) => {
 
   const handleLogoutClick = async () => {
     store.remove('TOKEN');
-    await keycloak.logout({ redirectUri: window.location.origin });
+
+    const siteMinderHost = process.env.NODE_ENV === 'production' ? 'logon7' : 'logontest7';
```

Comment:
Can we use `APP_ENV === 'prod'`?

Reply:
It is on FE, thus APP_ENV must be empty. I know what you mean, so I'll try to find a workaround.